### PR TITLE
YONK-372: Upgrade edx solutions discussion forum service fork to Eucalyptus

### PR DESCRIPTION
**Pull request by msaqib52**
This PR rebases edx-solutions discussion forum service fork to Eucalyptus.

@ziafazal would you please take a look at this?

### CODE REVIEW

**Comment by ziafazal** [comment]
@douglashall this PR has changes after rebasing solutions fork of cs_comment_service with Eucalyptus named release in upstream. Would you be able to review it?

**Comment by douglashall** [comment]
@ziafazal Anything worth highlighting in this changeset? Did you have to make extensive changes due to the rebase or was it straightforward?

**Comment by msaqib52** [comment]
@douglashall There were no extensive changes, rebase was straightforward and went smoothly.

File: spec/api/comment_thread_spec.rb [hunk 1]
```diff
@@ -565,7 +565,6 @@ def test_unicode_data(text)
 
       it "marks thread as read and confirms its value on returned response" do
         user = create_test_user(123)
-        thread = CommentThread.first
```
Comment (douglashall) [inline spec/api/comment_thread_spec.rb]: It seems strange that we are removing this variable declaration, but still referencing it in the lines below. Can you explain this change?

**Comment by douglashall** [comment]
One comment about a change we made to one of the tests. Otherwise, 👍 .

File: spec/api/comment_thread_spec.rb (hunk 1 above)
Comment (msaqib52) [inline spec/api/comment_thread_spec.rb]: we have a [helper method](https://github.com/edx-solutions/cs_comments_service/blob/ee9539c43f05de82fe299ada0b95f0900f35f530/spec/api/comment_thread_spec.rb#L453) at the start of this group which provides this variable for each test in this group, so no need of this variable declaration here

**Pull request merged**